yowsup/demos/jsonpost/layer.py

```diff
@@ -0,0 +1,71 @@
+from yowsup.layers.interface import YowInterfaceLayer, ProtocolEntityCallback
+from yowsup.layers.network   import YowNetworkLayer
+
+import datetime, sys, json, time, requests, os
```

Comment:
This would add unnecessary yowsup dependency on requests module, just for a demo